src/repo/learnocaml_process_exercise_repository.ml

```diff
@@ -147,6 +147,11 @@ let main dest_dir =
     match !exercises_index with
     | Some exercises_index -> exercises_index
     | None -> !exercises_dir / "index.json" in
+  let exercises_path =
+    dest_dir / Filename.dirname (Learnocaml_index.exercise_path "foo") in
```

Comment:
Why "foo"?

Reply:
any non-empty string would do, since we extract the dirname. `foo` is I believe a standard place-holder for such cases. I agree this is not very pretty, and we could either check for an existing exercise, or export the dirname directly from `Learnocaml_index`.